kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,11 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String S3_PROXY_HOST = "s3.proxy.host";
```

Comment:
Yes, sorry if it wasn't clear, I implied that splitting/parsing of a single string would be required in order to extract the arguments.